app/views/private_messages/_message.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="ma">
+  <% @ticket.private_messages.each do |private_message| %>
+    <% unless private_message.message.blank?%>
+      <div class="panel">
+        <span class="label secondary">
+          <a href="mailto:<%= private_message.user.email %>"><%= private_message.user.email %></a>  - said:
+        </span>
+        <div class="panel callout radius">
+          <%= private_message.message %>
+        </div>
+        <span class="label secondary"> on <%= private_message.created_at.strftime("%B %d, %Y %H:%M %p") %></span>
```

Comment:
This line can be removed if we fix the following line, because that will include the time as well.